js/repl/ReplOptions.js

```diff
@@ -501,6 +501,19 @@ class ExpandedContainer extends Component<Props, State> {
                 >
                   Built-ins
                 </LinkToDocs>
+                <select
+                  value={envConfig.corejs}
+                  className={styles.envPresetSelect}
+                  onChange={this._onEnvPresetSettingChange("corejs")}
+                  disabled={
+                    !envConfig.isEnvPresetEnabled ||
+                    !envConfig.isBuiltInsEnabled ||
+                    runtimePolyfillState.isEnabled
+                  }
+                >
+                  <option value="2">core-js 2</option>
+                  <option value="3">core-js 3</option>
```

Comment:
I feel like it is worthy to add
```html
<option value="3.6">core-js 3.6</option>
```
or simply replace core-js 3 by core-js 3.6 because 3.6 has more features support than core-js 3.0

Reply:
Oh I didn't know this makes a difference.